tests/basictests/dsp-operator.sh

```diff
@@ -157,9 +218,13 @@ test_metrics
 fetch_runs
 create_pipeline
 verify_pipeline_availabilty
+create_experiment
+verify_experiment_availabilty
 create_run
+create_experiment_run
 verify_run_availabilty
 check_run_status
+delete_experiment
```

Comment:
It would be better to move this below, delete_runs
as the runs in the experiment part may cause unknow trouble.

Reply:
Done, moved it below delete_runs.